.github/workflows/builds.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build binaries
+
+on: push
+
+permissions:
```

Comment:
I didn't know that this could appear at the toplevel!  Interesting.

Reply:
I've usually used `permissions: {}` for this in the past, but I've also been setting it per-job, so I don't know if it's the same...